Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -114,29 +114,23 @@ internal static void SetupLocalConnection()
         {
             if (LogFilter.Debug) Debug.Log("Client Connect Local Server");
 
-            RegisterSystemHandlers(true);
-
-            connectState = ConnectState.Connected;
+            // truly connect to the local server
+            RegisterSystemHandlers(true); // IMPORTANT: true for host mode!
```

Comment:
comments on previous line please.  
```suggestion
            // IMPORTANT: true for host mode!
            RegisterSystemHandlers(true);
```